docs/tech-preview.rst

```diff
@@ -1,7 +0,0 @@
-Tech previews
-=============
```

Comment:
so people can check what the current list of feature of tech preview. otherwise they do not know whether we just forgot to add such section or we did not foget and simply have no feature in tech preview. explicit is better than implicit.
Also think of yourself in the future and ease potential manual steps you will need to take. Are you going to add this https://github.com/pulp/pulp_container/pull/1287#discussion_r1202615902 and https://github.com/pulp/pulp_container/pull/1287#discussion_r1202617087 every time you get a feature in tech preview? And similarly, are you going to remove these 2 every time when you take out features from tech preview? :)